keyboards/massdrop/ctrl/keymaps/via/keymap.c

```diff
@@ -0,0 +1,140 @@
+/* Made by InonL (@InonL), June 2022
+ * Based on the default keymap, added two blank layers for VIA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum ctrl_keycodes {
+    U_T_AUTO = SAFE_RANGE, //USB Extra Port Toggle Auto Detect / Always Active
```

Comment:
If you use the "USER00-USER15" keycodes here, you can assign custom keycodes in the via json, so they are still accessible from VIA. 
```suggestion
    U_T_AUTO = USER00, //USB Extra Port Toggle Auto Detect / Always Active
```

Reply:
I wasn't aware of that, thanks!
I've commited the suggested change.